src/DataBus/obsoletes-v4.cs

```diff
@@ -0,0 +1,30 @@
+﻿#pragma warning disable 1591
+using System;
+using NServiceBus.DataBus;
+
+namespace NServiceBus
+{
+    public static partial class ConfigureAzureDataBus
+    {
+        [ObsoleteEx(
+            Message = "Authentication mechanism are supported by setting a BlobContainerClient. For simple cases, use ConnectionString",
```

Comment:
We're deprecating Managed Identity configuration API. Customers that used it won't go to the connection string method 🙂 How about this?

```suggestion
            Message = "Managed Identity authentication is supported by configuring Azure databus with a BlobContainerClient using '.BlobContainerClient()' configuration API.",
```